site/src/pages/tutorials/thrift/blog/implement-delete.mdx

```diff
@@ -0,0 +1,196 @@
+Check that you see the test is passed.
+
+## 5. Add the Documentation service
+
+This time, we'll add Armeria's [Documentation service](/docs/server-docservice).
+The Documentation service automatically creates documentation of your service methods, as well as providing means to test out the methods.
+
+1. In the `newServer()` method, add a <type://DocService> and a request example for [creating blog posts](/tutorials/thrift/blog/implement-create),
+  using <type://DocServiceBuilder#exampleRequests(Class,String,Iterable)>. Feel free to add more examples for other service methods.
+  ```java filename=Main.java
+  import com.linecorp.armeria.server.docs.DocService;
+  import example.armeria.blog.thrift.CreateBlogPostRequest;
+  ...
+  private static Server newServer(int port) throws Exception {
+      ...
+      final CreateBlogPostRequest exampleRequest = new CreateBlogPostRequest()
+              .setTitle("Example title")
+              .setContent("Example content");
+      final DocService docService = DocService
+              .builder()
+              .exampleRequests(List.of(new BlogService.createBlogPost_args(exampleRequest)))
+              .build();
+      ...
+  }
+  ```
+2. In the `newServer()` method, add the <type://DocService> to our server builder.
+  ```java filename=Main.java highlight=5
+  private static Server newServer(int port) throws Exception {
+      ...
+      return Server.builder()
+               .http(port)
+               .service("/thrift", tHttpService)
+               // You can access the documentation service at http://127.0.0.1:8080/docs.
+               // See https://armeria.dev/docs/server-docservice for more information.
+               .serviceUnder("/docs", docService)
+               .build();
+  }
+  ```
+
+3. (Optional) To access the Documentation service result easily, edit the log message in the `main()` method.
+  ```java filename=Main.java highlight=2
+  public static void main(String[] args) throws Exception {
+      ...
+      logger.info("Server has been started. Serving DocService at http://127.0.0.1:{}/docs",
+                  server.activeLocalPort());
+  }
+  ```
+4. Now, [re-run the server](/tutorials/thrift/blog/run-server-and-client#2-run-the-server).
+
+  The server and services are launched successfully if you see this message.
+  ```bash
+   Server has been started. Serving DocService at http://127.0.0.1:8080/docs
+  ```
+
+## 6. Check the DocService page
+
+Let's test and call our service operations using Armeria's Documentation service.
+
+1. Click the URL `http://127.0.0.1:8080/docs` from the log message or open up the URL on a web browser.
+
+  If you see the Document service page, you've successfully launched the <type://DocService> and server.
+
+  ![](../../../../images/tutorial_blogservice_thrift_docservice_start.png)
+
+2. Click the **CreateBlogPost()** method link in the left panel. You can make calls to the method from the page.
```

Comment:
The most recent changes in `DocService` require that the `Debug` button be clicked to enter the debug menu.
<img width="1680" alt="Screen Shot 2023-05-16 at 6 22 40 PM" src="https://github.com/line/armeria/assets/8510579/b9b215fb-9418-48d6-9190-5cc6eb9c0d38">

I don't think we need a screenshot, but I think it would be nice if we can briefly mention it.

i.e.
```suggestion
2. Click the **CreateBlogPost()** method link in the left panel. You can make calls to the method by clicking on the `Debug` button on the top right.
```